rust/android/Cargo.toml

```diff
@@ -12,10 +12,8 @@ android_logger = "0.11"
 log = "0.4"
 rustlib = { path = "../lib" }
 
-[dependencies.bellman]
-version = "0.13.0"
-default-features = false
-features = ["groth16"]
+[target.'cfg(target_arch = "i686")'.dependencies]
```

Comment:
I feel like something like this should work, since the error only comes in through `rayon` which is only an optional `multicore` dependency:

https://github.com/zkcrypto/bellman/blob/9bb30a7bd261f2aa62840b80ed6750c622bebec3/Cargo.toml#L30

But haven't gotten it to work..   it makes sense to me that we'll have to specify that we don't want `multicore` all the way down the depedency tree.